src/components/AutoUpdateTime.js

```diff
@@ -0,0 +1,102 @@
+import {View} from 'react-native';
```

Comment:
I'd say we should add a description to this as per the checklist guidelines. I was just tagged in this issue and it took me a little while to figure out why we needed this and not just directly use moment in the details page.

Reply:
how about this?
```
/**
 * The component that display user's local time and update it at the top of every minute
 */
```

Reply:
I'd add something about _why_ we need this component and can't just do it inline like we used to.

Reply:
Thanks @luacmartins . How about this one:
Just realized the way I wrote description is even harder than the way I fixed the issue 😂 
```
/**
 * The component that displays the user's local time and updates it at the top every minute.
 * This keeps the time up to date and prevents redundant renderings.
 */
``

Reply:
I think that still just describes _what_ it does. Maybe something like the description below would do:

```
Displays the user's local time and updates it every minute. The time auto-update logic is extracted to this component to avoid re-rendering a more complex component, e.g. DetailsPage.
 ```

Reply:
Thank you. I updated the PR!